aws/sdk-adhoc-test/build.gradle.kts

```diff
@@ -46,6 +48,7 @@ val allCodegenTests = listOf(
             ,
             "codegen": {
                 "includeFluentClient": false,
+                "enableNewSmithyRuntime": "${getSmithyRuntimeMode()}"
```

Comment:
are you intentionally passing the string middleware here? I'd expect this to be true/false

Reply:
It's a quad state: middleware, both defaulting to middleware, both defaulting to orchestrator, orchestrator